src/features/cartography/infrastructure/presentation/pages/cartography/cartography.presenter.ts

```diff
@@ -200,7 +200,12 @@ export class CartographyPresenter {
 
   public geocodeAddress$(addressToGeocode$: Observable<string>): Observable<Coordinates> {
     return addressToGeocode$.pipe(
-      switchMap((address: string): Observable<Coordinates> => this.geocodeAddressUseCase.execute$(address))
+      switchMap(
+        (address: string): Observable<Coordinates> =>
+          this.geocodeAddressUseCase.execute$(address).pipe(
+            catchError((): Observable<never> => EMPTY)
+          )
+      )
```

Comment:
À tester pour réduir le nesting :
```suggestion
      switchMap(
        (address: string): Observable<Coordinates> =>
          this.geocodeAddressUseCase.execute$(address)
      ),
      catchError((): Observable<never> => EMPTY)
```

Reply:
Je sais que ça ne marche pas.
Ce n'est pas équivalent car il faut que le catchError évite à geocodeAddressUseCase.execute$(address) de se résoudre